src/SASViyaApiClient.ts

```diff
@@ -1238,10 +1249,17 @@ export class SASViyaApiClient {
   private async pollJobState(
     postedJob: any,
     etag: string | null,
-    accessToken?: string
+    accessToken?: string,
+    pollOptions?: PollOptions
```

Comment:
We could have a default value here for `pollOptions` which has `MAX_POLL_COUNT` and `POLL_INTERVAL`.

Reply:
done